string pdfilename
bool showInterface
---
int8 resultcode